src/qt/guiutil.cpp

```diff
@@ -768,23 +768,66 @@ QString formatDurationStr(int secs)
     return strList.join(" ");
 }
 
+void sortLocaleAware(QStringList &sList)
+{
+    std::sort(sList.begin(), sList.end(), [](const QString &s1, const QString &s2){
+        return s1.localeAwareCompare(s2) < 0;
+    });
+}
+
 QString formatServicesStr(quint64 mask)
 {
     QStringList strList;
 
     for (const auto& flag : serviceFlagsToStr(mask)) {
+        if (flag == "NETWORK_LIMITED"){
+        strList.append(QString::fromStdString("LIMITED"));
+        }
+        else
+        {
         strList.append(QString::fromStdString(flag));
+        }
+        sortLocaleAware(strList);
     }
 
     if (strList.size())
-        return strList.join(" & ");
+        return strList.join(" ");
```

Comment:
This seems less clear

Reply:
This change allows the services string in the detail view to break.
The ampersands are extraneous (imo). 

![Screen Shot 2020-12-02 at 8 51 38 PM](https://user-images.githubusercontent.com/152159/100953280-4206e980-34e0-11eb-8182-6a835d71d226.png)